### PR TITLE
Add admin.updateAccountEamil

### PR DESCRIPTION
**Pull request by dholms**
Holding off on adding in user email updates because there are some security issues to take into account there

### CODE REVIEW

File: packages/pds/src/services/account/index.ts [hunk 1]
```diff
@@ -145,6 +145,17 @@ export class AccountService {
...
   }
 
+  async updateEmail(did: string, email: string) {
+    const res = await this.db.db
+      .updateTable('user_account')
+      .set({ email })
+      .where('did', '=', did)
+      .executeTakeFirst()
+    if (res.numUpdatedRows < 1) {
+      throw new UserAlreadyExistsError()
```
Comment (devinivy) [inline packages/pds/src/services/account/index.ts:155]: I think this error is probably something more like "user doesn't exist".  Maybe we can even get rid of the `getAccount()` check in the route.

File: packages/pds/src/services/account/index.ts [hunk 2]
```diff
@@ -145,6 +145,17 @@ export class AccountService {
     await sequenceHandleUpdate(this.db, did, handle)
   }
 
+  async updateEmail(did: string, email: string) {
+    const res = await this.db.db
```
Comment (devinivy) [inline packages/pds/src/services/account/index.ts:149]: Should we do some minimal form of email validation?  Just poked-around quickly at registration and didn't see any, though hmm 🤔

File: packages/pds/src/services/account/index.ts [hunk 3]
```diff
@@ -145,6 +145,17 @@ export class AccountService {
...
+      .updateTable('user_account')
+      .set({ email })
```
Comment (devinivy) [inline packages/pds/src/services/account/index.ts:151]: I think we may want to normalize the email to lowercase as we do elsewhere:
```suggestion
      .set({ email: email.toLowerCase() })
```

File: packages/pds/tests/account.test.ts [hunk 4]
```diff
@@ -168,6 +168,36 @@ describe('account', () => {
     ])
   })
 
+  it('allows administrative email updates', async () => {
+    await agent.api.com.atproto.admin.updateAccountEmail(
+      {
+        account: handle,
+        email: 'alice-new@test.com',
```
Comment (devinivy) [inline packages/pds/tests/account.test.ts:175]: Could toss some caps in here to test the normalization if we add that to the backend.

File: packages/pds/src/services/account/index.ts (hunk 1 above)
Comment (dholms) [reply packages/pds/src/services/account/index.ts:155]: ah yup lol. i did a hack copy-paste job

File: packages/pds/src/services/account/index.ts (hunk 3 above)
Comment (dholms) [reply packages/pds/src/services/account/index.ts:151]: ah good call 👍

File: packages/pds/tests/account.test.ts (hunk 4 above)
Comment (dholms) [reply packages/pds/tests/account.test.ts:175]: 👍

**Pull request merged**